docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The Dafny Language and Software Verification System
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+---
+<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+<script type="text/x-mathjax-config">
+        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ["\\(","\\)"]], displayMath: [ ["$$","$$"], ["\\[","\\]"] ]
+        }});
+</script>
```

Comment:
Do we foresee using any of those?

Reply:
MathJax is certainly used. I have a separate Issue to cleanup other html that is no longer used.

Reply:
In the index?

Reply:
In all files under docs